src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./style.css">
+    <title>Leaderboard</title>
+</head>
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+  <main>
+    <section class="left">
+      <div class="section-head">
+        <h2>Recent Scores</h2>
+        <button type="button" class="refresh">Refresh</button>
+      </div>
+      <div class="section-body">
+        <table class="table">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Score</th>
+            </tr>
+          </thead>
+          <tbody id="scores">
+          </tbody>
+      </table>
+      </div>
+      <button type="button" class="reset">Reset</button>
+    </section>
+    <section class="right">
+      <div class="section-head">
+        <h2>Add Your Score</h2>
+      </div>
+      <p class="success-message"></p>
+      <div class="section-body end">
+        <form id="c-form">
+          <label for="name">Name</label>
+          <input id="name" class="userdetails" type="text" minlength="2" placeholder="Your name" required>
+          <label for="Score">Score</label>
+          <input id="Score" class="userdetails" type="number" min="0" max="100" placeholder="Your score" required>
+          <button type="submit">Submit</button>
```

Comment:
- [ ] When I click on the submit button, I got this error: 


![Screenshot from 2022-05-13 11-55-22](https://user-images.githubusercontent.com/63087491/168270007-c523e91f-4730-4695-a87c-6b4f7db95ef8.png)

Please fix it :pray: